coralproject/talk
# [CORL-802] Wrap message box text properly

- Need to set the message content width so that the text knows how to wrap. 
- Also accounts for when the icon is present to set width to the correct value so it doesn't shift past the edge of the message box container.

## What does this PR do?

Fixes the text wrapping for message boxes in the stream on small screens.

Makes this:
<img width="232" alt="image" src="https://user-images.githubusercontent.com/5751504/71215350-b2d3d800-2274-11ea-9841-af758dfe8907.png">

Do this instead:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/5751504/71215297-946ddc80-2274-11ea-8af0-caa19188e973.png">

## How do I test this PR?

- Use Chrome or Safari to test this, Firefox isn't dumb and renders this properly without this fix
- Load up the stream as a mod
- Go to the Configure tab
- Enable `Enable message box for this story`
- Paste in the following or similar text with a link:

```
Have we missed your favourite London power-dining spot? What about in our other cities? The best tips entered in our super-quick survey (see link) will be published in FT Globetrotter soon. Or simply join in the conversation in the comments below.

https://survey.ft.com/jfe/form/SV_3P3lQwzqNYRMN6d
```

- You can choose to use an icon or not, it should work for both
- Open your dev tools and enable `Responsive Design Mode`
- Switch to a small screen type (I recommend trying iPhone 5, it's great for completely destroying supposedly responsive layouts)
- Should work

NOTE: on Firefox if you don't see iPhone 5 listed, use the `Edit List...` option in the device drop down and add it to your active list of devices
